float64 scaling_factor
float64 resolution_map
float64 resolution_grid
int64 grid_width
int64 grid_height
float64[] data